reanalysis/html_builder.py

```diff
@@ -82,10 +82,13 @@ def __init__(self, results: str | dict, panelapp: str, pedigree: Ped):
 
         # Use config to find CPG-to-Seqr ID JSON; allow to fail
         seqr_path = get_cohort_seq_type_conf().get('seqr_lookup')
-        self.seqr = {}
+        self.seqr: dict = {}
 
         if seqr_path:
-            self.seqr = read_json_from_path(seqr_path, default=self.seqr)
+            self.seqr: dict[str, str] = read_json_from_path(  # type: ignore
```

Comment:
I'd add the type annotation (str,str) on the line above, probably helps some typers

Reply:
good spot